deploy/zfs-operator.yaml

```diff
@@ -86,7 +528,7 @@ rules:
     resources: ["leases"]
     verbs: ["get", "watch", "list", "delete", "update", "create"]
   - apiGroups: ["*"]
-    resources: ["zfsvolumes"]
+    resources: ["zfsvolumes", "zfssnapshots"]
```

Comment:
above code is adding a few new CRDs from CSI, should they be added into the RBAC bindings.

Reply:
they are already part of RBAC bindings, I already has snapshotter role in the zfs operator yaml, which had those CRDs.